pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+_Please tick all applicable boxes, fill in the **bold** fields, and remove these instructions from your PR._
```

Comment:
I didn't think about that, typing an x isn't too much of a problem so I think you're right for keeping it how it is.